Makefile

```diff
@@ -44,7 +44,7 @@ help: ## Display this help.
 ##@ Development
 
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName="storageos-api-manager" webhook paths="./..." output:crd:artifacts:config=config/crd/bases output:webhook:artifacts:config=config/webhook/bases
```

Comment:
Not sure it matters here, but the operator will name the role `storageos:api-manager`.

Reply:
Answer same as : https://github.com/storageos/api-manager/pull/93#discussion_r711997010